package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "cuttle",
   "description": "The deepest card game under the sea",
-  "version": "4.8.1",
+  "version": "4.8.2",
```

Let's use a minor version since this is a new feature. The rule has been fuzzy, but the general idea has been:
1) patch for bug fixes, docs, style, and refactoring 
2) minor version for features
3) major version for sweeping architectural changes
```suggestion
  "version": "4.9.0",
```